mkdocs.yml

```diff
@@ -11,6 +11,8 @@ markdown_extensions:
   - admonition
   - pymdownx.details
   - pymdownx.superfences
+  - toc:
```

Comment:
?

Reply:
This is to limit the table of contents to a depth 3, so only #header1, ##header2 and ###header3 show up on the navigation.